README.md

```diff
@@ -7,7 +7,7 @@ data sources such as databases or web services via batching and caching.
 [![Build Status](https://travis-ci.org/facebook/dataloader.svg)](https://travis-ci.org/facebook/dataloader)
 [![Coverage Status](https://coveralls.io/repos/facebook/dataloader/badge.svg?branch=master&service=github)](https://coveralls.io/github/facebook/dataloader?branch=master)
 
-A port of the "Loader" API originally developed by [@schrockn][] at Facebook in
```

Comment:
@yujiangshui It's the correct word. `dataloader` was ported from Facebook internal PHP code.